warehouse/accounts/__init__.py

```diff
@@ -212,3 +213,6 @@ def includeme(config):
     config.add_periodic_task(
         crontab(minute="0", hour=4), batch_update_email_domain_status
     )
+    config.add_periodic_task(
+        crontab(minute="0", hour="*"), unverify_emails_with_expired_domains
```

Comment:
Should we move this to N minutes after the `batch_update_email_domain_status` task runs? that will make "time to remediation" N minutes, won't it?